createdisk.sh

```diff
@@ -74,7 +74,7 @@ EOF
 cat crio-wipe.service | ${SSH} core@${VM_IP} "sudo tee -a /etc/systemd/system/crio-wipe.service"
 
 # Preload routes controller
-${SSH} core@${VM_IP} -- "sudo crictl pull quay.io/crcont/routes-controller:${image_tag}"
+${SSH} core@${VM_IP} -- "sudo podman pull quay.io/crcont/routes-controller:${image_tag}"
```

Comment:
Fwiw, the `right now` part of the commit log is a bit worrying, sounds as if it's going to change any time in the future ^^